### PR TITLE
Post Getting Started Updates_Nov 30, 2021

### PR DESCRIPTION
**Pull request by mikki-stacey**
## Description

Include a high-level description of the issue your pull request fixes; include only one issue per pull request. Please also link to the GitHub issue it resolves.

## Additional information

Include any of the following (as necessary): 

* Relevant research and support documents
* Type of content review needed: stylistic or copy editing
* Screenshot images
* Notes

Before you hit Submit, make sure you’ve done whichever of these applies to you:

- [ ] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [ ] Run `npm test` and make sure the tests for the files you have changed have passed.
- [ ] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [ ] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -434,7 +310,14 @@ We’ve designed the design system to support older and newer browsers through [
 
 ## Accessibility
 
-The design system also meets the [WCAG 2.0 AA accessibility guidelines](https://www.w3.org/TR/WCAG20/) and conforms to the standards of [Section 508 of the Rehabilitation Act](http://www.section508.gov/). We’re happy to answer questions about accessibility — email us for more information.
+The design system also meets the [WCAG 2.0 AA accessibility guidelines](https://www.w3.org/TR/WCAG20/) and conforms to the standards of [Section 508 of the Rehabilitation Act](http://www.section508.gov/).
+
+We use the following tools to ensure USWDS is accessible:
+- ANDI (https://www.ssa.gov/accessibility/andi/help/install.html).
+- Axe core (https://www.deque.com/axe/).
+- Axe dev tools (https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US).
```
Comment (mejiaj) [inline README.md:318]: You can make these links by using markdown in the following format `[text](url)`.

```suggestion
- [ANDI](https://www.ssa.gov/accessibility/andi/help/install.html).
- [Axe core](https://www.deque.com/axe/).
- [Axe dev tools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US).
```

**Review by mejiaj — COMMENTED** [review]
Looks good! But we need to find a solution for SASS Requirements section.

You can view what the page would like like on the doc's site [here →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/jm-test-updated-readme/documentation/developers/).

I've also updated the links to point to new sections in the [Maturity Model page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/jm-test-updated-readme/maturity-model/#level-3-use-uswds-code).

**Review by thisisdano — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -134,126 +130,6 @@ We offer both the CSS and the JavaScript in two versions — minified and un-min
...
-```
-> 2%
-last 2 versions
-IE 11
-not dead
-```
-
-> Note: **media query sorting** is no longer required as of USWDS 2.5.0. We stopped sorting media queries with [csso](https://github.com/css/csso) in USWDS 2.5.1 because it wasn't outputting as expected. While both the minified and unminified CSS files are modestly larger as a result: `268 KB` unsorted vs. `259 KB` sorted, our testing indicates that once the files are compressed server side with gzip, the unsorted CSS is actually smaller: `36 KB` unsorted and gzipped vs. `38 KB` sorted and gzipped. As a result, we recommend that teams do not use media query sorting at this time.
-
-We recommend using a **minifier** like [csso](https://github.com/css/csso) to compress your final compiled CSS and **sourcemaps** like [`gulp-sourcemaps`](https://www.npmjs.com/package/gulp-sourcemaps) to keep track of the location of all the source Sass for easier debugging.
```
[the remark this answers is not in the document]
Comment (thisisdano) [reply README.md:245]: Let's reinstate it for now. But I think we should add a "Requirements" section to the **How to Use USWDS** section of the site and add this there.

**Review by thisisdano — CHANGES_REQUESTED** [review]
Let's make the change James suggested and then we can merge.

**Comment by mikki-stacey** [comment]
James suggestion was incorporated as part of this PR in the site repo: https://github.com/uswds/uswds-site/pull/1370/files

**Review by mejiaj — APPROVED** [review]
Looks good, thank you @mikki-stacey!

**Review by mejiaj — APPROVED** [review]

**Review by thisisdano — APPROVED** [review]

**Pull request merged**